extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -119,6 +125,40 @@
 
     }
 
+    @ConfigGroup
+    public static class Logout {
+
+        /**
+         * The logout endpoint at the application. If provided, the application is able to initiate the logout through this
+         * endpoint in conformance with the OpenID Connect RP-Initiated Logout specification.
+         */
+        @ConfigItem
+        Optional<String> path;
```

Comment:
If you agree with the `logout_path` proposal above, then `path` should likely be renamed to `localPath` may be, not 100% sure :-)

Reply:
I'm not sure. The doc change you mentioned is already stating it is a relative path at the application.

Reply:
@pedroigor That is fine, lets keep it as `path`, what about though that property which will let uses manually configure the OIDC logout path ?

Reply:
If we have for others, sure, let's provide also an option for this one at the same level.

Reply:
@pedroigor Sure that would be fine